Wrappers/Python/cil/optimisation/operators/GradientOperator.py

```diff
@@ -132,6 +132,23 @@ def adjoint(self, x, out=None):
         """            
         return self.operator.adjoint(x, out=out)
 
+
+    def calculate_norm(self):
+        
+        r""" Returns the analytical norm of the GradientOperator.
+                
```

Comment:
Wouldn't it make sense to have at least a citation and/or a math formula here?

Reply:
Do we have a docstring example with maths in it I can refer to?